base/doc/ltnews38.tex

```diff
@@ -299,8 +299,10 @@ \section{Changes to packages in the \pkg{amsmath} category}
 \section{Changes to packages in the \pkg{graphics} category}
 
 \section{Changes to packages in the \pkg{tools} category}
+\subsection{\pkg{longtable}: correct p-column definition}
 
-
+In general the \pkg{longtable} implementation follows 
+the \pkg{array} usage but the package didn't take over a change made 1992 in \pkg{array} which adjusted the handling of the strut inserted at the begin of p-columns. As a consequence there are a number of inconsistencies in the output of p-columns between \pkg{tabular} and \pkg{longtable}. This has been corrected, \pkg{longtable} uses now for the strut the same definition as \pkg{array}. \githubissue{1128}
```

Comment:
may add a few newlines :-) but more importantly, maybe add a note that the old behavior can be obtained using an explicit rollback version as @davidcarlisle mentioned